<html>
<head>
  <title>Lactococcus lactis</title>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta charset="utf-8">
  <link rel="icon" type="image/png" href="/assets/icons/chassidex.png" />
  <link rel="stylesheet" type="text/css" href="/assets/styles/hostpage2.css">
  <script src="/assets/scripts/sidebar.js"></script>
</head>
<body onload="generateSidebar()">
  <header>
    <a href="/">
      <img src="/assets/icons/chassidex-text.png">
    </a>
  </header>
  <nav id="sidebar"></nav>
  <section id="content">
    <p class="title">Lactococcus lactis</p>
    <h1>About</h1>
<p>Members of the Lactococcus genus are non-motile, catalase-negative and occur singly, in pair or in chain. They are Gram-positive, facultative anaerobe, and do not form spores. Based on their metabolic stability, resistance to bacteriophage, and the ability to produce unique compounds often from amino acid catabolism, having several fermentation pathways, and due to the phosphoenolpyruvate-phosphotransferase system their most important purpose is its property to manufacture dairy product such as cheese and milk. This ILI403 strain of Lactococcus lactis is diploid. The parent strain of IL1403 has a citrate permease plasmid and is able to metabolize citrate. Lactis subsp. lactis IL1403 is a plasmid-cured strain that lacks lactose transport and utilization genes, the cell envelope protease, a citrate transporter, restriction-modification systems, and other plasmid-encoded features. Dairy industries continue to improve the activities and effectiveness of L. lactis by manipulating its environment and cell behavior.</p>
<h2>References:</h2>
<ul>
<li><a href="https://genome.jgi.doe.gov/laccr/laccr.home.html">https://genome.jgi.doe.gov/laccr/laccr.home.html</a></li>
<li><a href="http://www.uniprot.org/proteomes/UP000002196">http://www.uniprot.org/proteomes/UP000002196</a></li>
<li><a href="https://microbewiki.kenyon.edu/index.php/Lactococcus_lactis">https://microbewiki.kenyon.edu/index.php/Lactococcus_lactis</a></li>
</ul>
<h1>Growth Characteristics</h1>
<p>Lactococci grows at 28-30 °C in M17 broth (Merck) and is supplemented with either 0.5% (wt/vol) d-glucose (GM17) or 0.5% (wt/vol) d-xylose (XM17) as the sole carbon source for growth of the plasmid-free dairy strain of ILI403. Maintain the stock culture at −85 °C in M17 broth supplemented with 20% (v/v) glycerol. L. lactis IL1403 expressed growth rate in A. thaliana lysate (ATL) was found to be (0.63 h−1). IL1403 reached cell density upto 4.9 × 107 CFU/ml. 5% CO2 in the surrounding atmosphere is ideal.</p>
<p>References:</p>
<ul>
<li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2812979/">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2812979/</a></li>
<li><a href="http://jb.asm.org/content/197/2/371.full">http://jb.asm.org/content/197/2/371.full</a></li>
<li><a href="https://www.atcc.org/Products/All/19435.aspx#culturemethod">https://www.atcc.org/Products/All/19435.aspx#culturemethod</a></li>
</ul>
<h1>Maintenance Protocols</h1>
<ol>
<li>Media and growth conditions for Lactococcus lactis :<br>
<a href="http://classrooms.tacoma.k12.wa.us/wilson/vrehberg/documents/download/agar_multitype_basic_protocols.pdf?id=145903">http://classrooms.tacoma.k12.wa.us/wilson/vrehberg/documents/download/agar_multitype_basic_protocols.pdf?id=145903</a></li>
</ol>
<h1>Transformation Protocols</h1>
<ol>
<li><p>Efficient transformation:<br>
<a href="http://onlinelibrary.wiley.com/doi/10.1002/jobm.200610297/pdf">http://onlinelibrary.wiley.com/doi/10.1002/jobm.200610297/pdf</a></p>
</li>
<li><p>Electrotransformation of L. lactis.:<br>
<a href="https://www.nature.com/protocolexchange/protocols/287#/procedure">https://www.nature.com/protocolexchange/protocols/287#/procedure</a></p>
</li>
</ol>
<h1>Other Protocols</h1>
<ol>
<li>VBEx (Vector backbone exchange) procedure:<br>
<a href="https://www.nature.com/protocolexchange/protocols/287#/procedure">https://www.nature.com/protocolexchange/protocols/287#/procedure</a></li>
</ol>
<h1>Biobrick Parts</h1>
<ol>
<li><p>BBa_K137009: <a href="https://openwetware.org/wiki/IGEM:Caltech/2008/Project/Vitamins">https://openwetware.org/wiki/IGEM:Caltech/2008/Project/Vitamins</a></p>
</li>
<li><p>BBa_K137011: <a href="https://openwetware.org/wiki/IGEM:Caltech/2008/Project/Vitamins">https://openwetware.org/wiki/IGEM:Caltech/2008/Project/Vitamins</a></p>
</li>
<li><p>BBa_K137005: <a href="https://openwetware.org/wiki/IGEM:Caltech/2008/Project/Vitamins">https://openwetware.org/wiki/IGEM:Caltech/2008/Project/Vitamins</a></p>
</li>
<li><p>BBa_K137006: <a href="https://openwetware.org/wiki/IGEM:Caltech/2008/Project/Vitamins">https://openwetware.org/wiki/IGEM:Caltech/2008/Project/Vitamins</a></p>
</li>
</ol>
<h1>Vectors</h1>
<ol>
<li><p>pSpeedET: <a href="http://dnasu.org/DNASU/GetCloneDetail.do?cloneid=326951">http://dnasu.org/DNASU/GetCloneDetail.do?cloneid=326951</a></p>
</li>
<li><p>pSEUDO: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3187151/">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3187151/</a></p>
</li>
</ol>
<h1>Metabolic Models</h1>
<ol>
<li>Genome scale flux model: <a href="https://bmcmicrobiol.biomedcentral.com/articles/10.1186/1471-2180-5-39">https://bmcmicrobiol.biomedcentral.com/articles/10.1186/1471-2180-5-39</a></li>
</ol>

  </section>
</body>